From: Felipe Sateler <fsateler@debian.org>
Date: Fri, 6 Jun 2014 13:15:18 -0400
Subject: Fix implementation of aligned_allocator::construct<U>.

Fixes a build failure with gcc >= 4.9, because it defines __cplusplus >= 201103L.

A typo, apparently. This patch can be dropped in the next upstream release.
Index: supercollider/server/supernova/utilities/malloc_aligned.hpp
===================================================================
--- supercollider.orig/server/supernova/utilities/malloc_aligned.hpp	2014-09-11 09:15:20.399357542 +0100
+++ supercollider/server/supernova/utilities/malloc_aligned.hpp	2014-09-11 09:15:20.399357542 +0100
@@ -243,7 +243,7 @@
     template< class U, class... Args >
     void construct(U * p, Args&& ... args)
     {
-        ::new(p) T(std::forward<Args>(args)...);
+        ::new(p) U(std::forward<Args>(args)...);
     }
 #endif

